tox/_pytestplugin.py

```diff
@@ -292,6 +292,24 @@ def fnmatch_lines(self, lines2):
 @pytest.fixture
 def initproj(request, tmpdir):
     """ create a factory function for creating example projects. """
+    # Constructed folder/file hierarchy examples:
```

Comment:
Let's make this part of the docstring, otherwise 🆗

Reply:
done